packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -362,16 +358,13 @@ export const DetailsTile = ({
             <DetailsProgressInfo track={track} />
           ) : null}
           <View style={styles.buttonSection}>
-            {isGatedContentEnabled &&
-              !doesUserHaveAccess &&
-              premiumConditions &&
-              trackId && (
-                <DetailsTileNoAccess
-                  trackId={trackId}
-                  premiumConditions={premiumConditions}
-                />
-              )}
-            {!isGatedContentEnabled || doesUserHaveAccess ? (
+            {!doesUserHaveAccess && premiumConditions && trackId && (
```

Comment:
Tern-around is fair play.
(That's my weird word play to say you caught me on a ternary last PR and it's my turn muahaha) 😁

Reply:
lawl got me